[E:runtime][2018-04-19 09:19:23 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-19 09:19:23 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:No such file or directory
[E:runtime][2018-04-19 09:19:23 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-19 09:19:23 @user.server.startup:52] apply id error:No such file or directory
[E:runtime][2018-04-19 09:19:23 @user.server.startup:52] apply id error:No such file or directory
[E:runtime][2018-04-19 09:19:24 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-19 09:19:24 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-19 09:19:24 @./script/server/login.lua:81] login listen client success
[E:runtime][2018-04-19 09:19:25 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:19:25 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:20:11 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 09:20:11 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:12 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:12 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:12 @user.server.startup:70] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:13 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:20:13 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:20:27 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:27 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:27 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:28 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:20:28 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:20:44 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:44 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:44 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:20:45 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:20:45 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:20:45 @./script/server/login.lua:90] login listen client success
[E:runtime][2018-04-19 09:21:01 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:21:01 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:21:01 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:21:01 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 09:21:02 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:21:02 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:21:03 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:21:03 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:21:04 @./script/server/login.lua:89] login listen client success
[E:runtime][2018-04-19 09:25:48 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:25:48 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:25:48 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:25:48 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 09:25:48 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 09:25:49 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:25:49 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:25:49 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:25:50 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:25:50 @./script/server/login.lua:105] login listen client success
[E:runtime][2018-04-19 09:25:50 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:26:39 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 09:26:39 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:26:39 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:26:39 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 09:26:40 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:26:40 @user.server.startup:70] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:26:40 @user.server.startup:70] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 09:26:41 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:26:41 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 09:26:41 @./script/server/login.lua:96] login:all agent ready:35	33
[E:runtime][2018-04-19 09:26:41 @./script/server/login.lua:105] login listen client success
[E:runtime][2018-04-19 11:43:34 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-19 11:43:34 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-19 11:43:34 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:No such file or directory
[E:runtime][2018-04-19 11:43:34 @user.server.startup:52] apply id error:No such file or directory
[E:runtime][2018-04-19 11:43:34 @user.server.startup:52] apply id error:No such file or directory
[E:runtime][2018-04-19 11:43:35 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-19 11:43:35 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-19 11:43:35 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:No such file or directory
[E:runtime][2018-04-19 11:43:36 @./lualib/event.lua:73] ./lualib/import.lua:19: user.handler.agent_handler:118: '}' expected (to close '{' at line 115) near 'scene_pos'
stack traceback:
	[C]: in function 'util.load_script'
	./lualib/import.lua:19: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	./script/server/agent.lua:67: in upvalue 'func'
	./lualib/event.lua:51: in function <./lualib/event.lua:50>
[E:runtime][2018-04-19 11:43:36 @./lualib/event.lua:73] ./lualib/import.lua:19: user.handler.agent_handler:118: '}' expected (to close '{' at line 115) near 'scene_pos'
stack traceback:
	[C]: in function 'util.load_script'
	./lualib/import.lua:19: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	./script/server/agent.lua:67: in upvalue 'func'
	./lualib/event.lua:51: in function <./lualib/event.lua:50>
[E:runtime][2018-04-19 11:43:36 @./script/server/login.lua:96] login:all agent ready:4	3
[E:runtime][2018-04-19 11:43:36 @./script/server/login.lua:105] login listen client success
[E:runtime][2018-04-19 11:43:57 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 11:43:57 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 11:43:57 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:43:57 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:43:57 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 11:43:58 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 11:43:58 @user.server.startup:70] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 11:43:58 @user.server.startup:70] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 11:43:59 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 11:43:59 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 11:44:00 @./script/server/login.lua:96] login:all agent ready:10	9
[E:runtime][2018-04-19 11:44:00 @./script/server/login.lua:105] login listen client success
[E:runtime][2018-04-19 11:44:29 @./lualib/event.lua:352] user.module.scene_server:16: attempt to call a nil value (field 'fetch_fighter')
stack traceback:
	user.module.scene_server:16: in local 'info'
	./lualib/event.lua:320: in function <./lualib/event.lua:313>
	[C]: in function 'xpcall'
	./lualib/event.lua:350: in function <./lualib/event.lua:344>
	[C]: in method 'dispatch'
	./lualib/event.lua:261: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-19 11:44:29 @./lualib/event.lua:352] user.module.scene_server:16: attempt to call a nil value (field 'fetch_fighter')
stack traceback:
	user.module.scene_server:16: in local 'info'
	./lualib/event.lua:320: in function <./lualib/event.lua:313>
	[C]: in function 'xpcall'
	./lualib/event.lua:350: in function <./lualib/event.lua:344>
	[C]: in method 'dispatch'
	./lualib/event.lua:261: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-19 11:44:59 @./lualib/event.lua:352] user.module.scene_server:16: attempt to call a nil value (field 'fetch_fighter')
stack traceback:
	user.module.scene_server:16: in local 'info'
	./lualib/event.lua:320: in function <./lualib/event.lua:313>
	[C]: in function 'xpcall'
	./lualib/event.lua:350: in function <./lualib/event.lua:344>
	[C]: in method 'dispatch'
	./lualib/event.lua:261: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-19 11:44:59 @./lualib/event.lua:352] user.module.scene_server:16: attempt to call a nil value (field 'fetch_fighter')
stack traceback:
	user.module.scene_server:16: in local 'info'
	./lualib/event.lua:320: in function <./lualib/event.lua:313>
	[C]: in function 'xpcall'
	./lualib/event.lua:350: in function <./lualib/event.lua:344>
	[C]: in method 'dispatch'
	./lualib/event.lua:261: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-19 11:45:22 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 11:45:22 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:22 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:22 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 11:45:22 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:23 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:23 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:23 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:24 @./script/server/login.lua:96] login:all agent ready:14	13
[E:runtime][2018-04-19 11:45:24 @./script/server/login.lua:105] login listen client success
[E:runtime][2018-04-19 11:45:26 @./lualib/event.lua:73] ./lualib/import.lua:19: stack overflow
stack traceback:
	./lualib/import.lua:19: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.module.scene_server:5: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	...
	user.module.scene_server:5: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.handler.scene_handler:2: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	./script/server/scene.lua:56: in upvalue 'func'
	./lualib/event.lua:51: in function <./lualib/event.lua:50>
[E:runtime][2018-04-19 11:45:26 @./lualib/event.lua:73] ./lualib/import.lua:19: stack overflow
stack traceback:
	./lualib/import.lua:19: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.module.scene_server:5: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	...
	user.module.scene_server:5: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.handler.agent_handler:5: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	./script/server/agent.lua:67: in upvalue 'func'
	./lualib/event.lua:51: in function <./lualib/event.lua:50>
[E:runtime][2018-04-19 11:45:26 @./lualib/event.lua:73] ./lualib/import.lua:19: stack overflow
stack traceback:
	./lualib/import.lua:19: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	user.module.scene_server:5: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	...
	user.module.scene_server:5: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	user.module.scene_user:8: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	user.handler.agent_handler:5: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	./script/server/agent.lua:67: in upvalue 'func'
	./lualib/event.lua:51: in function <./lualib/event.lua:50>
[E:runtime][2018-04-19 11:45:46 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:46 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:46 @user.server.startup:52] apply id error:Connection refused
[E:runtime][2018-04-19 11:45:46 @user.server.startup:70] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:47 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:47 @user.server.startup:70] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-19 11:45:48 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 11:45:48 @./script/server/agent.lua:69] agent start success
[E:runtime][2018-04-19 11:45:48 @./script/server/login.lua:96] login:all agent ready:20	19
[E:runtime][2018-04-19 11:45:48 @./script/server/login.lua:105] login listen client success
